CHANGELOG.next.asciidoc

```diff
@@ -77,6 +77,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 
 *Filebeat*
 
+- Fixed various Cisco FTD parsing issues. {issue}16863[16863] {pull}16889[16889]
```

Comment:
It's better to add the entries at the end of the group to keep chronological order